creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -143,11 +143,34 @@ class SnapbackSM {
     // The interval when SnapbackSM is fired for state machine jobs
     this.snapbackJobInterval = this.nodeConfig.get('snapbackJobInterval') // ms
 
+    this.stateMachineQueueJobMaxDuration = this.snapbackJobInterval * 5
+
+    // Wipe all redis keys matching pattern `bull:state-machine*` (legacy and new)
```

Comment:
this gets rid of all the legacy data that we had when we used to assign each bull queue a unique name with timestamp to make it easier to debug + keep redis state clean

also removes new state as well, before queueing - feels dangerous but don't actually think it is, since it's before we create the new queue. we really never ever want to have any holdover state from previous queue runs so i think this is fine

Reply:
what's the intention here? would we need to do this if we had a uniquely named queue?

Reply:
yeah this is mostly to clean up all the redis state from when we had uniquely named queues

Reply:
got it. i'm open to putting this in the code but seems better being run as a one off script

Reply:
yeh, thats fine too but also its kind of fine to run it every time since it will clear all state from previous runs which is good

Reply:
ripped it out. couldn't get it working determinsitically, didn't make sense to ship this and risk any state corruption. will instead just run this on our nodes with cmd `redis-cli --scan --pattern 'bull:state-machine-*' | xargs redis-cli DEL`